Federation/FederationTypeInterceptor.cs

```diff
@@ -149,8 +150,8 @@ private void CompleteReferenceResolver(ObjectTypeDefinition typeDef)
         ITypeCompletionContext completionContext,
         DefinitionBase? definition)
     {
-        if (completionContext.IsQueryType == true &&
-            definition is ObjectTypeDefinition objectTypeDefinition)
+        if (definition is ObjectTypeDefinition objectTypeDefinition &&
```

Comment:
`IsQueryType` method was removed from `ITypeCompletionContext` so we no longer can easily verify whether it is a root query type or not. Workaround is to look at the `IDescriptorContext.Options.QueryTypeName`.... but this values DOES NOT get populated unless users provide custom name.

In vast majority of cases, users should use default `Query` type. We can preset this in federation extension. If users have custom query type names, they will have to explicitly provide the value.